proposals/75:upgrade-16/probe-submission/probeZcfBundle-plan.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "probeZcfBundle",
+  "script": "probeZcfBundle.js",
+  "permit": "probeZcfBundle-permit.json",
+  "bundles": [
+    {
+      "entrypoint": "@agoric/vats/src/vat-zoe.js",
+      "bundleID": "b1-db507028460b0c9ad9b16a1e1094b4cd09cfbdc1779c992b42d828745788d98393ca4740599ac4b2e3a6f4bd8d9018aad6d509107eced9253fe7d3c0085709a7",
+      "fileName": "/home/connolly/.agoric/cache/b1-db507028460b0c9ad9b16a1e1094b4cd09cfbdc1779c992b42d828745788d98393ca4740599ac4b2e3a6f4bd8d9018aad6d509107eced9253fe7d3c0085709a7.json"
```

Comment:
do these local file paths on your machine matter?

Reply:
They don't affect execution.

I guess cleaning them up for looks is worthwhile...